acquia/acquia_cms
# ACMS-1098: Initial API setup wizard

**Motivation**
As an user who has access to admin tour dashboard page, I want to see the options for different headless option use cases so that I can choose one of the options from given use cases.

**Proposed changes**
- Updates Headless module description.
- Adds Robust and Headless options to the Tour dashboard and Wizard (Currently, nothing happens when either option is checked other than the option being stored in the database / config. See #1073 and #1062.

**Alternatives considered**
N / A

**Testing steps**
- Check out branch.  
- Complete a fresh site install
- Enable the **Acquia CMS Headless** module.
- Confirm that Next JS or any of it's dependencies have been installed.
- Enable Robust and save.  
- Confirm message and confirm that Robust API is still checked on reload.
- Enable headless and save.  
- Confirm message and confirm that headless is still checked on reload.
- Steps can be repeated to disable option either together or separately.

**Merge requirements**

**Additional Notes**
- The description text for the Robust and Headless form options are WIP and may need to be updated either here during review or via a future ticket.
- Does not currently include any functional phpunit tests, see #1101 for separate task.
